### PR TITLE
Various fixes & improvements to image results

### PR DESCRIPTION
**Pull request by JamesWrigley**
I've already deployed this to the beta module.

### CODE REVIEW

**Comment by takluyver** [comment]
Other than that, LGTM. I've changed the PR title to reflect that it's not just the one fix any more.

File: damnit/ctxsupport/ctxrunner.py [hunk 1]
```diff
@@ -479,15 +479,12 @@ def summarise(self, name):
             return data
         elif isinstance(data, Figure):
             # For the sake of space and memory we downsample images to a
-            # resolution of 35 pixels on the larger dimension.
+            # resolution of THUMBNAIL_SIZE pixels on the larger dimension.
             image_shape = data.get_size_inches() * data.dpi
             zoom_ratio = min(1, THUMBNAIL_SIZE / max(image_shape))
             return figure2png(data, dpi=(data.dpi * zoom_ratio))
         elif is_array and data.ndim == 2 and self.ctx.vars[name].summary is None:
-            from scipy import ndimage
```
[the remark this answers is not in the document]
Comment (JamesWrigley) [reply damnit/ctxsupport/ctxrunner.py:487]: Yep, removed in 9c08abf.

[the remark this answers is not in the document]
Comment (takluyver) [reply damnit/ctxsupport/ctxrunner.py:487]: Thanks! Can you update the [constraints file](https://github.com/European-XFEL/DAMNIT/blob/master/.github/dependabot/constraints.txt) too so we don't get pointless dependabot PRs updating it?

[the remark this answers is not in the document]
Comment (JamesWrigley) [reply damnit/ctxsupport/ctxrunner.py:487]: Ah yes, very good point. Removed it in a831c7d, and also removed a mention of it from the docs.

**Pull request merged**